l5kit/l5kit/data/map_api.py

```diff
@@ -11,7 +12,14 @@
 ENCODING = "utf-8"
 
 
+class InterpolationMethod(IntEnum):
+    INTER_METER = 0  # fixed interpolation at a given step in meters
+    INTER_ENSURE_LEN = 1  # ensure we always get the same number of elements
+
+
 class MapAPI:
+    TR_FACES_COLORS = ["red", "green", "yellow"]
```

Comment:
I would either define the `InterpolationMethod` as a string list too, or change this `TR_FACES_COLORS` to an enum too (you can get the string value by the `.name()` method from the enum.